src/transactions/utils/validation.js

```diff
@@ -28,6 +44,18 @@ export const validatePublicKey = publicKey => {
 	return true;
 };
 
+export const validateSignature = signature => {
+	const signatureBuffer = hexToBuffer(signature);
+	if (signatureBuffer.length !== 64) {
+		throw new Error(
+			`Signature ${
+				signature
+			} length differs from the expected 64 bytes for a public key.`,
+		);
+	}
+	return true;
```

Comment:
This return value is not parallel.
If we `return true`, i think it's better to have `return false`